src/mcts/params.cc

```diff
@@ -313,10 +313,10 @@ void SearchParams::Populate(OptionsParser* options) {
   options->Add<ChoiceOption>(kHistoryFillId, history_fill_opt) = "fen_only";
   options->Add<FloatOption>(kMovesLeftMaxEffectId, 0.0f, 1.0f) = 0.0f;
```

Comment:
I also think we should set `max-effect=0.1` per default, so to activate MLH one only needs to set the threshold and nothing else.

Reply:
Agreed